test/test-v8debugapi.js

```diff
@@ -43,6 +45,67 @@ function stateIsClean(api) {
   return true;
 }
 
+function validateVariable(variable) {
```

Comment:
Earlier you mentioned moving this validation into the source as opposed to the tests. Is there a reason to hold off on doing that?